integration_tests/mockApis/curiousApi.ts

```diff
@@ -0,0 +1,17 @@
+import { SuperAgentRequest } from 'superagent'
+import { stubFor } from './wiremock'
+
+const stubCuriousApiPing = (): SuperAgentRequest =>
+  stubFor({
+    request: {
+      method: 'GET',
+      urlPattern: '/curious-api/ping',
+    },
+    response: {
+      status: 200,
+      headers: { 'Content-Type': 'text/plan;charset=UTF-8' },
+      body: 'pong',
```

Comment:
The Curious health ping endpoint returns `pong` ! It's even documented that way in their swagger spec! 🤣

Reply:
lol!